tests/vmi_multus_test.go

```diff
@@ -1026,17 +1026,24 @@ func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAdd
 		&expect.BExp{R: console.RetValue("0")},
 	}, 15)
 	Expect(err).ToNot(HaveOccurred(), "Failed to configure address %s for interface %s on VMI %s", interfaceAddress, interfaceName, vmi.Name)
+	Expect(setUpInterface(vmi, interfaceName)).To(Succeed(), "Failed to set interface %s up on VMI %s", interfaceName, vmi.Name)
+}
+
+func configSRIOVInterface(vmi *v1.VirtualMachineInstance, interfaceMac, interfaceAddress string) {
+	err, interfaceName := getSRIOVInterfaceName(vmi)
+	Expect(err).NotTo(HaveOccurred(), "should get sriov name")
 
-	setIfaceUpCmd := fmt.Sprintf("ip link set %s up\n", interfaceName)
+	cmdCheck := fmt.Sprintf("ip addr add %s dev %s\n", interfaceAddress, interfaceName)
 	err = console.SafeExpectBatch(vmi, []expect.Batcher{
 		&expect.BSnd{S: "\n"},
 		&expect.BExp{R: console.PromptExpression},
-		&expect.BSnd{S: setIfaceUpCmd},
+		&expect.BSnd{S: cmdCheck},
 		&expect.BExp{R: console.PromptExpression},
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BExp{R: console.RetValue("0")},
 	}, 15)
-	Expect(err).ToNot(HaveOccurred(), "Failed to set interface %s up on VMI %s", interfaceName, vmi.Name)
+	Expect(err).ToNot(HaveOccurred(), "Failed to configure address %s for SRIOV interface on VMI %s", interfaceAddress, vmi.Name)
+	Expect(setUpInterface(vmi, interfaceName)).To(Succeed(), "Failed to set SRIOV interface up on VMI %s", vmi.Name)
```

Comment:
arent u doing it twice now ? both here an on line https://github.com/kubevirt/kubevirt/pull/4001/files#diff-ac233e621be1a1855fe741915d860d91484c7f061fc42beb213f4f9e0a756f86R1029 ?